html/changelogs/AutoChangelog-pr-30531.yml

```diff
@@ -0,0 +1,4 @@
+author: "Kerbin_Fiber"
```

Comment:
Since you have a changelog in your PR description you don't need this file, it will be automatically generated for you.